util/etcd/etcd.go

```diff
@@ -0,0 +1,368 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package etcd
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	"go.etcd.io/etcd/clientv3"
+	"go.etcd.io/etcd/etcdserver/etcdserverpb"
+	"google.golang.org/grpc"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const etcdTimeout = 2 * time.Second
+
+// exponential backoff for etcd operations, values taken from kubeadm
```

Comment:
```suggestion
// exponential backoff for etcd operations, values taken from kubeadm.
```

Consistency with the other comments and the rest of the repository.